framework/test/unit/modules/nft/endpoint.spec.ts

```diff
@@ -419,21 +429,37 @@ describe('NFTEndpoint', () => {
 				},
 			});
 
-			const expectedSupportedCollection = {
-				collectionIDs: supportedCollections.map(collection =>
-					collection.collectionID.toString('hex'),
-				),
-			};
+			await expect(endpoint.getSupportedCollectionIDs(context)).resolves.toEqual({
+				supportedCollectionIDs: [Buffer.concat([chainID, collectionID]).toString('hex')],
+			});
+		});
 
-			await expect(endpoint.getCollectionIDs(context)).resolves.toEqual(
-				expectedSupportedCollection,
-			);
+		it('should return an array with supported collection IDs for native chains when chain is equal to ownChainID', async () => {
```

Comment:
The supportedNFTs store never stores entries with key equal to `OWN_CHAIN_ID`, see this sentence in [Properties](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0052.md#properties-2) section for the Supported NFTs substore in the LIP:
```
Since the native NFTs are always supported, no entry with key OWN_CHAIN_ID is added to the substore.
```
So, this test does not make sense and it should be removed or written differently (not sure I understand what was the original intention of the test so I don't have any recommendations how to refactor it).

Reply:
I was testing the this added logic

`# Add native collections
for collectionID in set([getCollectionID(nftID) for nftID in NFTSubstore.keys()]):
    supportedCollectionIDs.append(OWN_CHAIN_ID.hex() + collectionID.hex())`

Reply:
OK, maybe I understood the test wrong because I got the impression it operates with Supported NFTs substore, while LIP prohibits to have `OWN_CHAIN_ID` key in that substore ("supported" always refers only to non-native NFTs).

Reply:
I've removed the test as you rightly point out that `native NFTs are always supported`. This test is already covered

Reply:
OK. 
The possible confusion we all have with this originates in the fact that, when speaking of supported NFTs (or NFT collections, or tokens), we always mean both native and non-native objects. However, both the NFT and Token module LIP (and I'm not too happy with this) define "supported", i.e. "that which is in Supported NFTs store", _only for non-native objects_. 
That's why you need to add native collections in the code separately from "supported" as stored in the Supported NFTs store.

Reply:
I am not sure where the logic related to NFT store data if "native collection id's" are correctly returned by the endpoint and that they are the uinque is already covered but for this particular PR I think we still need to have atleast one corresponding test to have a complete unit test coverage.